lib/MusicBrainz/Server/Edit/Annotation/Edit.pm

```diff
@@ -38,6 +38,7 @@ role {
                 id   => Int,
                 name => Str
             ]],
+            old_annotation_id => Optional[Nullable[Int]],
```

Comment:
Optional because old edits don't have it, nullable because many entities do not yet have an annotation, and the first addition will be undef.